docs/content/docs/dev/table/sql/queries/hints.md

```diff
@@ -279,6 +279,49 @@ SELECT /*+ NEST_LOOP(t1) */ * FROM t1 JOIN t2 ON t1.id = t2.id;
 SELECT /*+ NEST_LOOP(t1, t3) */ * FROM t1 JOIN t2 ON t1.id = t2.id JOIN t3 ON t1.id = t3.id;
 ```
 
+#### MULTI_JOIN
```

Comment:
Please add the Chinese version of this as well.

Reply:
I'll create a ticket for the community to pick it up since I have no Chinese knowledge to review the output of automatic translation